elasticsearch_dsl/_async/document.py

```diff
@@ -126,12 +162,18 @@ async def exists(cls, id, using=None, index=None, **kwargs):
         ``Elasticsearch.exists`` unchanged.
         """
         es = cls._get_connection(using)
-        return await es.exists(index=cls._default_index(index), id=id, **kwargs)
+        return bool(await es.exists(index=cls._default_index(index), id=id, **kwargs))
```

Comment:
Suppressing the low level client response here and returning a bool, which seems more clear to me.